### PR TITLE
[NON-MODULAR] Adds a crafting recipe for the double emergency oxygen tank

### PR DESCRIPTION
**Pull request by desertmote**
## About The Pull Request

A simple crafting recipe addition. Two engi tanks and sticky tape makes a double engi tank.

## Why It's Good For The Game

The double emergency tank is nice, it gives about 40 minutes of air. To my knowledge it is only attainable through maintenance spawn as of now.

## Changelog
:cl:
add: Adds a crafting recipe for the double emergency oxygen tank.
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
2 air is two times the good il allow it

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -1030,6 +1030,17 @@
 	result = /obj/item/pickaxe/improvised
 	category = CAT_MISC
 
+// Skryat addition start
```
Comment (Gandalf2k15) [inline code/datums/components/crafting/recipes.dm:1033]: Perhaps move this to a modular folder, no need for it to be here

Comment (desertmote) [reply code/datums/components/crafting/recipes.dm:1033]: will look into this

**Comment by Low3** [comment]
What about makeshift double emergency tank (non engie one). I think it will be a nice addition. Also adding crafting recepies is not great because bloat. Can you make it so when you have duct tape in offhand and click one tank with another it consumes few stacks of tape, deletes tanks and puts in hand double one. Also what about tanks contents? Do they get merged or overwritten? For such cases adding check that they both are empty would be nice. And ofcource result will be empty doubletank.

**Review by Gandalf2k15 — APPROVED** [review]
Honestly seems good to me.

**Pull request merged**